app/lib/actions/katello/environment/publish_repositories.rb

```diff
@@ -0,0 +1,50 @@
+module Actions
+  module Katello
+    module Environment
+      class PublishRepositories < Actions::EntryAction
+        middleware.use ::Actions::Middleware::RemoteAction
+
+        input_format do
+          param :id
+          param :name
+        end
+
+        #???? options content_type Katello::Repository::DOCKER_TYPE
+        def plan(env, options = {})
+          action_subject(env)
+          sequence do
+            concurrence do
+              env.repositories.each do |repository|
+                repository.save!
+                plan_action(::Actions::Katello::Repository::Update, repository, {})
+                plan_action(::Actions::Pulp::Repository::Refresh, repository)
```

Comment:
@jlsherrill @parthaa I'm just spamming actions here to try to get the the repo_registry_id to be updated in pulp. Any pointers of what to call to have that happen?

Reply:
I think I have this sorted out; will clean up code a bit since I think this double action is not needed.